apps/fluent-tester/src/FluentTester/TestComponents/ActivityIndicator/ActivityIndicatorTest.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import { useRef } from 'react';
+import { Text } from '@fluentui/react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { ACTIVITYINDICATOR_TESTPAGE } from './consts';
+import { View, StyleSheet, Animated, Easing, ViewProps } from 'react-native';
+
+const radius = 40; // from center to halfway border
+const border = 10; // line thickness
+const xydist = Math.sqrt(2)/2 * radius;
+
+// let R = big circle radius, from middle of circle to middle of border width = (width - borderRadius) / 2
+// borderRadius only refers to big circle radius
+// D = sqrt(2)/2 * R is distance from center to the middle of end circle
+// Top end circle:
+  // top: R - D - borderRadius
+    // top edge is already 1 borderRadius below the big circle
+  // left: R - D - borderRadius
+    // left edge is already 1 borderRadius to the right
+// Bottom end circle:
+  // top: R + D - 2 * borderRadius
+    // 1 borderRadius explained same as top end circle. Extra borderRadius since bottom end circle is 1 borderRadius below top end circle already.
+  // left: R - D - borderRadius
+    // same explanation as above
+
+const styles = StyleSheet.create({
+  semicircle: {
+    width: radius*2 + border, // from outside edge of circle on one side to the opposite side
+    height: radius*2 + border,
+    borderWidth: border,
+    borderColor: '#919191',
+    borderRadius: radius + border/2,
+    borderLeftColor: 'transparent',
+  },
+  endTop: {
+    width: border,
+    height: border,
+    borderRadius: border/2,
+    backgroundColor: '#919191',
+    top: radius - xydist - border,
+    left: radius - xydist - border,
+  },
+  endBottom: {
+    width: border,
+    height: border,
+    borderRadius: border/2,
+    backgroundColor: '#919191',
+    top: radius + xydist - 2*border,
+    left: radius - xydist - border,
+  }
+});
+
+
+export type ActivityIndicatorProps = ViewProps & {
+    animating?: boolean;
+    hidesWhenStopped?: boolean;
+}
+const ActivityIndicator = (props: ActivityIndicatorProps) => {
+  const spinAnimation = useRef(new Animated.Value(0)).current;
+  if (props.animating) {
```

Comment:
This should be in it's own package `@fluentui-react-native/activityindicator` rather than just a function in the test code. That would ensure people can actually import it and keep a separation of responsibility between the component and the test.
You can copy one of the existing packages in `packages/experimental/*` as a template and then change the names / files to be your component. I have some info on how to do that in our contribution guide https://github.com/microsoft/fluentui-react-native/blob/master/CONTRIBUTING.md

Reply:
Just added as its own package. Along the way, there were some extra steps that aren't on the contributing guide and it might be helpful to add those to the guide. (Deleting cache and running `yarn install` to load the package as a dependency for the app and some import/export tips.)

Reply:
Thanks =) If you have a list of those, we can definitely improve the the contribution guide :D 
Thanks for teasing the guide out through practice, haha